src/components/blogs/index.css

```diff
@@ -0,0 +1,40 @@
+.blogPhotoContainer img {
+  width: 80%;
+  text-align: center;
+  transition: width ease-in-out 0.5s;
+  margin: 20px 0;
+}
+
+div.blogPhotoContainer {
```

Comment:
`div.` prefix here seems unnecessary unless youre using blogphotocontainer in multiple contexts